pkgs/development/tools/rust/cargo-c/default.nix

```diff
@@ -13,16 +13,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "cargo-c";
-  version = "0.9.29";
+  version = "0.9.31";
 
   src = fetchCrate {
     inherit pname;
     # this version may need to be updated along with package version
-    version = "${version}+cargo-0.76.0";
-    hash = "sha256-Uy5Bm8WwN3jQO2btnV/ayxTlIJAe5q2FUvhxCCrn9U8=";
+    version = "${version}+cargo-0.78.0";
```

Comment:
just curious, does this mean that this release requires Cargo 0.78.0?

Reply:
Good question. I don't believe that's the case, but I will return with a conclusive answer when I can find time to investigate further.